README.md

```diff
@@ -114,22 +114,21 @@ After you're finished please remove all the comments and instructions!
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo Link](https://deploy-preview-31--rentbikes.netlify.app)
+- [Live Demo Link]
```

Comment:
_note: Since this PR does not include all the files, I'll include my frontend review comments here._

- [ ] Upon running the app according with your readme's instructions and with the required command `npm run dev`, I get these screens:

![image](https://user-images.githubusercontent.com/57421823/230682146-cc63c644-503f-4e05-aee0-ecd12ac2522f.png)

The console:

![image](https://user-images.githubusercontent.com/57421823/230682200-e7f63f0e-d41f-45b9-9143-316bc7c06bec.png)

Upon inspecting the code, I can see that in App.jsx there is this import:

![image](https://user-images.githubusercontent.com/57421823/230682301-7867aa23-10f4-41c7-9953-2650f3b52cd5.png)

However, at that location, the file does not exist:

![image](https://user-images.githubusercontent.com/57421823/230682355-c03ecbbb-0885-4950-8975-76a0f979767f.png)

Thus, the app refuses to run.  Was that file deleted by accident?  I'm reviewing based on the "test" branch, as requested on the review request.

Kindly fix the import so the app can be run properly. 👍 

If this is a bug, kindly fix it.  If it's not and your app requires special procedures to be run properly, kindly add those instructions to your readme, so all your users can run your app properly. 💪 

_note:  Since I couldn't run the app, I wasn't able to test the functionality and the styling.  This will be done in the next review._

- [ ]  Please, kindly refactor the functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry:

![image](https://user-images.githubusercontent.com/57421823/230682794-8dadf856-109a-40c2-8e0f-c4e3f511a024.png)

- [ ] Good job adding frontend tests!, however, they are not passing for me:

![image](https://user-images.githubusercontent.com/57421823/230683256-e2bf7720-38b6-4cb9-85d8-0f15127a54d3.png)

This is the error output that I'm getting for each of the failed tests:

![image](https://user-images.githubusercontent.com/57421823/230683359-b74a66a9-2528-4d30-bb1d-08a8afd271d2.png)

If I scroll down a little for more details:

![image](https://user-images.githubusercontent.com/57421823/230683400-f4843628-43db-405e-8b31-737291cfd0c2.png)

If this is a bug in the tests, kindly fix it.  If it's not and your app requires special procedures to be run properly, kindly add those instructions to your readme, so all your users can run your tests properly. 💪